cli/delete.go

```diff
@@ -55,9 +55,11 @@ func deleteProjectCommand(t *core.Timetrace) *cobra.Command {
 				Key: key,
 			}
 
-			if !askForConfirmation() {
-				out.Info("Record NOT deleted.")
-				return
+			if !confirmed {
+				if !askForConfirmation() {
+					out.Info("Project NOT deleted.")
+					return
+				}
 			}
```

Comment:
Wouldn't that be a bit nicer code? Avoid stacked ifs if possible and feasible.
```suggestion
			if !confirmed && !askForConfirmation() {
				out.Info("Project NOT deleted.")
				return
			}
```

Reply:
Yes that makes sense. In fact, I was merely mindlessly copying the if-logic from the `deleteRecordCommand` and did not even think that there was a cleaner way to do it. I will submit another commit with an edit to `deleteRecordCommand` as well.

Reply:
While discussing code style, I also refactored the `if options.Revert` checks to remove the `else` statement for cleaner code, following the advice from [https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88](url).

Please have a look at the new commit and let me know if that's alright!